spec/spec_helper.rb

```diff
@@ -18,3 +26,32 @@
   config.order = :random
   Kernel.srand config.seed
 end
+
+def hatchet_app
```

Comment:
It looks like `hatchet_app` and `client` moved here because they are now used in multiple test files, yeah? If so, I wonder if we can/should move them into a `spec/support/*` module, and then include that specifically in the test files that need them? Doing so seems a nice balance of having them easily available, but also not being global, and seeming to come from no where. Thoughts?